request_handler.py

```diff
@@ -93,7 +97,11 @@ def do_POST(self):
         content_len = int(self.headers.get('content-length', 0))
         post_body = json.loads(self.rfile.read(content_len))
         response = ''
+<<<<<<< HEAD
+        (resource, id) = self.parse_url(self.path)
+=======
```

Comment:
this is a repeat of line 96

Reply:
I accepted the incoming change from Main